godotopenxrlynx/README.md

```diff
@@ -0,0 +1,28 @@
+# Godot OpenXR Lynx Android loader
+
+Binaries taken from Lynx at:
+https://portal.lynx-r.com/downloads/item/1/download/latest
+
+# AndroidManifest changes
+
+Add this to your manifest to enable OpenXR and Ultraleap's hand tracking:
+
+```
+<uses-feature android:name="android.hardware.vr.headtracking" android:required="true" android:version="1"/>
+<uses-permission android:name="org.khronos.openxr.permission.OPENXR"/>
+
+<queries>
+    <intent>
+        <action android:name="org.khronos.openxr.OpenXRRuntimeService"/>
+    </intent>
+    <provider android:authorities="org.khronos.openxr.runtime_broker;org.khronos.openxr.system_runtime_broker" />
+    <package android:name="com.ultraleap.tracking.service"/>
+    <package android:name="com.ultraleap.openxr.api_layer"/>
+</queries>
```

Comment:
These can be added to the plugin's manifest, which through Android's manifest merging will automatically merge them with the generated app's manifest.

There are some limitations to the manifest merging logic which is why we've added some documentation in the `README` for the KHR loader, but for this loader it doesn't seem to be needed.

Reply:
Well that answers my questions :)
I've made the changes to the AndroidManifest but I haven't tested it yet. There is some vendor-specific thing related to hand tracking but this can be discussed in another thread.